src/gsi-ncdiag/gsi_ncdiag.py

```diff
@@ -1387,24 +1421,26 @@ def toIODAobs(self, OutDir, ObsBias, QCVars, TestRefs, clobber=True):
                     outdata[(value2, 'MetaData')] = self.var(key).astype(np.int32)
                 else:
                     outdata[(value2, 'MetaData')] = self.var(key).astype(np.float32)
-                VarDims[(value2, 'MetaData')] = ['nchans']
+                VarDims[(value2, 'MetaData')] = ['Channel']
                 if value2 in units_values.keys():
                     varAttrs[(value2, 'MetaData')]['units'] = units_values[value2]
             except IndexError:
                 pass
 
         # global attributes
-        globalAttrs["satellite"] = self.satellite
-        globalAttrs["sensor"] = self.sensor
+        globalAttrs["platform"] = str(wmo_satid[self.satellite])
```

Comment:
Should these be strings or integers in the attributes?

Reply:
@CoryMartin-NOAA Fixed the platform and sensor data type in global attributes from string to integer as you suggested.